drivers/pwm/Kconfig.pca9685

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2022 Nick Ward <nix.ward@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config PWM_PCA9685
+	bool "PCA9685 16-channel, 12-bit PWM Fm+ I2C-bus LED controller"
+	depends on I2C
+	help
```

Comment:
please use `DT_HAS_*` helpers, and `select I2C`

Reply:
Updated